server/sonar-server/src/main/java/org/sonar/server/user/ws/UserJsonWriter.java

```diff
@@ -93,7 +93,7 @@ public void write(JsonWriter json, @Nullable UserDto user) {
   }
 
   private void writeGroupsIfNeeded(JsonWriter json, Collection<String> groups, @Nullable Collection<String> fields) {
```

Comment:
same comment as above:
a UT should be added, right?
now System admin of default org can see when organizations are disabled